services/validation/src/ValidationService.ts

```diff
@@ -46,48 +45,20 @@ export class ValidationService implements IValidationService {
         return response;
     }
 
-    private findInputFiles(files: any): any {
-        const inputs: any = [];
-
-        if (files && files.files) {
-            // Case: <UploadedFile[]>
-            if (Array.isArray(files.files)) {
-                files.files.forEach((file: { data: any; }) => {
-                    inputs.push(file.data)
-                })
-                return inputs;
-
-                // Case: <UploadedFile>
-            } else if (files.files["data"]) {
-                inputs.push(files.files["data"]);
-                return inputs;
-            }
-
-            // Case: default
-            const msg = `Invalid file(s) detected: ${util.inspect(files.files)}`;
-            this.logger.error(msg);
-            throw new Error(msg);
-        } else if (files) {
-            const truffleFiles: any[] = this.attemptTruffleExtraction(files[0]);
+    private findInputFiles(files: {data: Buffer}[]): Buffer[] {
+        if (files.length === 1) {
+            const truffleFiles = this.attemptTruffleExtraction(files[0]);
```

Comment:
Is there a way to make this more generic, i.e. not mention "truffle" explicitly in the function name and in general search all files for "encapsulated" metadata?

Reply:
Sure, I'll make changes in that direction; just to note that a few more changes have been added to the pull request, but have not yet been published.